src/framework/audio/internal/worker/audioengine.cpp

```diff
@@ -137,22 +119,20 @@ unsigned int AudioEngine::sampleRate() const
     return m_sampleRate;
 }
 
-std::shared_ptr<IAudioBuffer> AudioEngine::buffer() const
+IAudioBufferPtr AudioEngine::buffer() const
 {
     ONLY_AUDIO_WORKER_THREAD;
     return m_buffer;
 }
 
-std::shared_ptr<IMixer> AudioEngine::mixer() const
+IMixerPtr AudioEngine::mixer() const
 {
-    ONLY_AUDIO_WORKER_THREAD;
```

Comment:
Why is this marker removed?
I would recommend creating the necessary markers and placing them as much as possible to identify errors

Reply:
I don't actually recall that, maybe it was just accidentally removed

Anyway, nice catch. Fixed in https://github.com/musescore/MuseScore/pull/8471/commits/1f4270c0615b7641b18b9f43c8f0026b35f7392c